src/main/java/dev/galacticraft/mod/events/GCEventHandlers.java

```diff
@@ -94,9 +95,10 @@ public static void onServerTick(MinecraftServer server) {
             // calculate frequency of meteors on current celestial body
             float atmospherePressure = celestialBody.value().atmosphere().pressure();
             float frequency = (atmospherePressure <= Mth.EPSILON) ? 5.0f : (atmospherePressure * 100.0f);
+            frequency *= 1.0f / Galacticraft.CONFIG.meteorSpawnMultiplier();
```

Comment:
Why don't you just use the `/=` assignment operator?

```suggestion
            frequency /= Galacticraft.CONFIG.meteorSpawnMultiplier();
```

Reply:
Oh, you should also be careful about dividing by zero. You may want to deal with this by setting a minimum value in `ConfigImpl.java`.